images/Dockerfile.ubuntu

```diff
@@ -41,50 +42,51 @@ FROM ${FLAVOR}-${FLAVOR_RELEASE}-upstream AS common
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update \
-  && apt-get install -y --no-install-recommends \
+  && apt-get install -y --no-install-recommends --no-install-suggests \
     gnupg \
     software-properties-common \
   && add-apt-repository ppa:oibaf/test \
   && apt-get update \
-  && apt-get install -y --no-install-recommends \
+  && apt-get install -y --no-install-recommends --no-install-suggests \
     conntrack \
     console-data \
     cloud-guest-utils \
-    coreutils \
+    # coreutils \
     cryptsetup \
     curl \
     debianutils \
     dosfstools \
-    dracut \
-    dracut-network \
     e2fsprogs \
     fdisk \
     file \
     gawk \
     gdisk \
-    grub2-common \
     haveged \
     iproute2 \
     iptables \
     iputils-ping \
     jq \
-    kbd \
+    # added in https://github.com/kairos-io/kairos/commit/d413f0ec9c420a4847105e8b531e948cf4ce3176
+    # kbd \
     krb5-locales \
     less \
-    lldpd \
+    # can I remove this? it brings perl
+    # lldpd \
     lvm2 \
-    nano \
+    # there's pico
+    # nano \
     nbd-client \
-    neovim \
+    # vim is even bigger, so maybe just use pico
+    # neovim \ 
     nohang \
     open-iscsi \
     open-vm-tools \
     openssh-server \
     parted \
     rsync \
-    shim-signed \
-    snapd \
-    snmpd \
+    # snapd \
+    # can I remove this? it brings perl
+    # snmpd \
     squashfs-tools \
```

Comment:
Do we need this? We used to do so when we supported squashfs images as we needed the squash tools in the image but I don't think we longer need them?

Reply:
Ah we may need it for dracut but we no longer ship images in squashfs but in the livecd and I think that has the modules already via luet package

Reply:
I don't think we need dracut with UKI

Reply:
ok, then I will move to the "grub" section so it's not installed for UKI images